tool/autoqa/schemaorg/manual-annotations.js

```diff
@@ -592,21 +594,16 @@ const STRING_FILE_OVERRIDES = {
     'org.schema.Restaurant:Restaurant_name': 'org.openstreetmap:restaurant',
     'org.schema.Person:Person_name': 'tt:person_full_name',
     'org.schema.Person:Person_address_addressLocality': 'tt:location',
-    'org.schema.Hotel:Hotel_geo': 'org.schema.Hotel:Hotel_address_addressLocality',
```

Comment:
I think this change goes back to the problem of not being able to specify distinct parameter files for each language. Do you need to do it this way? If not, I prefer to retain the old approach.

Reply:
You broke schema2qa experiment with this. We don't do filters on street/city in schema2qa. 
If you want to use this script to generate a different schema, you should make it an option for SPL experiments.

Reply:
Not sure what you mean by "broke", cause the tests were passing when changes were merged and it was reviewed by you and Giovanni.
Change it however you want. I will take a look at SPL experiments later.

Reply:
We do not do filters on streetAddress or addressLocality (city) in all the annotated data of schema2qa dataset. In fact, they are not even in the schema. Your changes broke that and make them appear in the schema and generate training data filtering on them instead of geo. It won't break any test but will get a very low accuracy on schema2qa eval and test.  

I suppose you modified those eval/test examples in SPL?

Reply:
I do have parameter files for streetAddress and addressLocality for each language, and I was generating new schemas for SPL experiments. Feel free to merge this, and I will test SPL with genie master. If it broke SPL, we can discuss how to change this later.

Reply:
I think the right solution is that all these overrides should be gated by the locale parameter, so you can generate a locale-specific schema file appropriate for SPL. Because using the English manifest might not be correct in other languages for other reasons too.

Reply:
Yeah, having one schema per language is a possible solution.